<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageLayerDialog</class>
 <widget class="QWidget" name="ImageLayerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Image layer settings</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="StyledFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="filenameLbl">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Image file:</string>
        </property>
        <property name="buddy">
         <cstring>filenameEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="filenameEdit"/>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="filenameBtn">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="StyledFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,1,3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Horizontal scroll speed:</string>
        </property>
        <property name="buddy">
         <cstring>xScrollEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Vertical scroll speed:</string>
        </property>
        <property name="buddy">
         <cstring>yScrollEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="UnitLineEdit" name="xScrollEdit"/>
      </item>
      <item row="1" column="1">
       <widget class="UnitLineEdit" name="yScrollEdit"/>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="EmbossWidgetSmall" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StyledFrame</class>
   <extends>QFrame</extends>
   <header>Widgets/styledframe.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>UnitLineEdit</class>
   <extends>QLineEdit</extends>
   <header>Widgets/unitlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>EmbossWidgetSmall</class>
   <extends>QWidget</extends>
   <header>Widgets/embosswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>filenameEdit</tabstop>
  <tabstop>filenameBtn</tabstop>
  <tabstop>xScrollEdit</tabstop>
  <tabstop>yScrollEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
